x/vesting/types/schedule.go

```diff
@@ -281,5 +281,5 @@ func AlignSchedules(
 		endQ += period.Length
 	}
 	endTime = Max64(endP, endQ)
-	return
+	return startTime, endTime
```

Comment:
@fedekunze Please could you explain the cognitive complexity error displayed on Ln 152?

func ConjunctPeriods(